refactoring/initial_two_classes.py

```diff
@@ -32,17 +32,30 @@ def add_person(self, name, age, job):
 
     def number_of_connections(self, name):
         """Find the number of connections that a person in the group has"""
-        pass
+        return len(self.connections[name])
 
     def connect(self, name1, name2, relation, reciprocal=True):
         """Connect two given people in a particular way.
         Optional reciprocal: If true, will add the relationship from name2 to name 1 as well
         """
-        pass
+        if not name1 in self.connections:
```

Comment:
Also could have a guard clause here to check that we know about these people
```suggestion
        if not self.contains(name1):
            raise ValueError(f"I don't know who {name1} is.")
        if not self.contains(name2):
            raise ValueError(f"I don't know who {name2} is.")
        if not name1 in self.connections:
```